unity-renderer/Assets/Scenes/InitialScene.unity

```diff
@@ -234,7 +234,7 @@ MonoBehaviour:
   useCustomContentServer: 0
   customContentServerUrl: http://localhost:1338/
   baseUrlMode: 3
-  customURL: https://play.decentraland.zone/?
+  customURL: http://localhost:8080/?ENABLE_WEB3=&
```

Comment:
maybe revert?

Reply:
Thanks for the catch! I must have misclicked during commit. Reverted